src/models/weatherData/wind.js

```diff
@@ -0,0 +1,8 @@
+export default class Wind {
+  constructor(inSightWeatherData) {
```

Comment:
I hadn't realised you were using all the weather data! I'd be tempted to make this take two arguments `inSightWindSpeedData` and `inSightWindData` to make it clear what HWS and WD mean but it's up to you!